Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -158,7 +158,7 @@ theorem uniformEmbedding_subtype_val {p : α → Prop} :
 
 theorem uniformEmbedding_set_inclusion {s t : Set α} (hst : s ⊆ t) :
     UniformEmbedding (inclusion hst) where
-  comap_uniformity := by rw [uniformity_subtype, uniformity_subtype, comap_comap]; rfl
+  comap_uniformity := by rw [uniformity_subtype, uniformity_subtype, Filter.comap_comap]; rfl
```

Comment:
Flagging to other reviewers: this is necessary as there is also `UniformSpace.comap_comap`, which comes in scope with the `open UniformSpace` above.